RageItalic/EasyOutlines
# disabled recaptcha, enabled back btn

EXPECTED FUNCTIONALITY:
You should now be able to get thesis ideas and an essay outline without going through and waiting for recaptcha v3. The one change is when you click get outline, you will be redirected to the stripe checkout page. 

**WARNING: We are using the live version of stripe now, so the 42424242... test card will no longer work. Payment will fail if you try this, which is absolutely expected.**

HOW TO TEST:
Normal flow of the app as usual. You can use your actual card to make a purchase if you would like to do so. BUT, in case you want to test for free, just lmk and I will change back to test key and redelpoy the edge function just so you can test before we push and payments are officially live. 

**TO NOTE: Once on the stripe checkout page, you can decide to go back and it should redirect to the thesis ideas page with the thesis options all there. Also, once you are on the essay outline page, you can click on the back button and should be able to go back to the thesis ideas page as well. BOTH these scenarios need to be tested before this goes live.**
